api/src/controllers/rss.js

```diff
@@ -112,17 +115,19 @@ exports.post = (req, res) => {
 			if (!feeds.feedUrls.length) {
 				return res
 					.status(404)
-					.send('We couldn\'t find any feeds for that RSS feed URL :(');
+					.send("We couldn't find any feeds for that RSS feed URL :(");
```

Comment:
by default, my `.prettierrc` always uses single ticks and then just escapes apostrophes. is this something that you'd like to enforce either way? if so, go ahead and pull this so we're not switching it back and forth every PR

Reply:
Yeah, I'm really not a fan of double quotes.